### ISSUE TITLE
rename repo to just "aid-app"

### ISSUE THREAD

**Issue by lowellbander** [dandelion-community/aid-app-monorepo#19]
@bluquar u cool w me doing this?

**Comment by bluquar** [comment]
Yep. It'll break the aws deployment though lol. Easiest fix is to edit the yaml files to include "monorepo" in the app and env names

**Comment by bluquar** [comment]
You can just change .github/workflows/deploy_to_eb.yaml to hardcode the names of the destination app and environment to “aid-app-monorepo-${branch}” instead of “${repo}-${env}”. This won’t break anything since the fact that “aid-app-monorepo” is used in the AWS app and env names arbitrarily / because I hardcoded it there. AWS doesn’t know/care about the name of the repo.

**Comment by bluquar** [comment]
This will no longer break any AWS deployments.
